stage1/common/types/pod.go

```diff
@@ -49,13 +76,39 @@ func (p *Pod) AppNameToImageName(appName types.ACName) types.ACIdentifier {
 	return image.Name
 }
 
-// LoadPod loads a Pod Manifest (as prepared by stage0) and
+// SaveRuntime persists just the runtime state. This should be called when the
+// pod is first created
```

Comment:
Does "create" here mean "prepared" or "run"? (Also, does this work for prepared pods?)